aibolit/ml_pipeline/ml_pipeline.py

```diff
@@ -116,7 +117,8 @@ def train_process():
         model_new = pickle.load(fid)
         print('Model has been loaded successfully')
         for x in X_test:
-            preds, importances = model_new.predict(x)
+            snippet = [i for i in x] + [randint(1, 200)]
```

Comment:
What randint(100) do?

Reply:
it's just a random number. New interface demands ncss at the end of snippter. But training dataset doesn't have ncss. We have only patterns. And that part of code relates to smoke testing after training a dataset